kolena/_api/v1/core.py

```diff
@@ -57,9 +57,6 @@ class DeleteRequest:
         id: int
 
 
-Model.LoadAllResponse.__pydantic_model__.update_forward_refs()  # type: ignore
```

Comment:
Do we not need this anymore? It looks like the migration guide states that this got renamed:
<img width="394" alt="Screenshot 2024-01-31 at 10 32 37" src="https://github.com/kolenaIO/kolena/assets/2481598/5e049240-b04c-47db-a576-ee97cc222ab0">

Reply:
Nothing seems to break when I remove it, so I see no reason to keep it. The reason I removed it here is not because it was renamed but because [Pydantic V2 dataclasses no longer have a `__pydantic_model__`](https://docs.pydantic.dev/latest/migration/#changes-to-dataclasses):

> Pydantic dataclasses no longer have an attribute `__pydantic_model__`, and no longer use an underlying `BaseModel` to perform validation or provide other functionality.